src/community/security/oauth2-openid-connect/pom.xml

```diff
@@ -38,11 +38,15 @@
       <artifactId>gs-sec-oauth2-core</artifactId>
       <version>${project.version}</version>
     </dependency>
-
+    <dependency>
+      <groupId>com.jayway.jsonpath</groupId>
+      <artifactId>json-path</artifactId>
+      <version>2.4.0</version>
```

Comment:
```suggestion
```

Reply:
No need for version, it is covered by `src/pom.xml` dependency management already.